spec/factories/kudos.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :kudo do
+    title { 'Kudo Title' }
+    content { 'Kudo Content' }
```

Comment:
try to make it more random, because from time to time you will be creating multiple kudos via factory - maybe use sequence or Faker for title and content

Reply:
Right, now every title and content will be more unique